examples/with_winit/src/lib.rs

```diff
@@ -453,6 +453,23 @@ enum UserEvent {
     HotReload,
 }
 
+#[cfg(target_arch = "wasm32")]
+fn display_error_message() -> Option<()> {
+    let window = web_sys::window()?;
+    let document = window.document()?;
+    let elements = document.get_elements_by_tag_name("body");
+    let body = elements.item(0)?;
+    let canvas = body.first_child()?;
+    // TODO: style the notice at least a little, maybe link?
+    let text = document.create_text_node(
+        "WebGPU is not enabled. Make sure your browser is updated to
+        Chrome M113 or another browser compatible with WebGPU.",
+    );
```

Comment:
You can (presumably) use `innerHTML` here to include an entire HTML string inline.

Reply:
Ah, good point. For some reason, I was going down the path of creating it using DOM manipulation, but that would be better.

Reply:
```rust
let div = document.create_element("div");
div.set_inner_html("HTML here...");
```
if you're wondering about the exact incantation.